site/editor.html

```diff
@@ -43,6 +43,12 @@
 			z-index: 10;
 		}
 
+		#icons {
```

Comment:
These should be tailwind classes. Also, it's not visible for me in chrome; I might have broken something 😬